scipy/fft/__init__.py

```diff
@@ -44,6 +44,25 @@
    dstn - N-D Discrete sine transform
    idstn - N-D Inverse discrete sine transform
 
+Chirp Z-transform (:mod:`scipy.fft.czt`)
+============================================
+
+.. module:: scipy.fft.czt
```

Comment:
This is the problem. You are documenting them as in this nested module, but they are imported directly into scipy.fft so they are expected to be documented there

Reply:
```suggestion
```

Reply:
Ok.  Why do the functions have descriptions in the `__init__` file when they are overwritten during doc generation?


```
   dct - Discrete cosine transform
   idct - Inverse discrete cosine transform
   dctn - N-D Discrete cosine transform
   idctn - N-D Inverse discrete cosine transform
```

becomes 

> dct(x[, type, n, axis, norm, overwrite_x, …]) | Return the Discrete Cosine Transform of arbitrary type sequence x.
> idct(x[, type, n, axis, norm, overwrite_x, …]) | Return the Inverse Discrete Cosine Transform of an arbitrary type sequence.
> dctn(x[, type, s, axes, norm, overwrite_x, …]) | Return multidimensional Discrete Cosine Transform along the specified axes.
> idctn(x[, type, s, axes, norm, overwrite_x, …]) | Return multidimensional Discrete Cosine Transform along the specified axes.


for instnace

Reply:
When accessing `scipy.fft.__doc__`, you will see these descriptions.